twine/__main__.py

```diff
@@ -20,12 +20,25 @@
 from twine import cli
 from twine import exceptions
 
+try:
+    import colorama
+except Exception:
```

Comment:
Should this be an `ImportError` or `Exception`? If `Exception` what are we catching here and why?

Reply:
In [pip/logging.py](https://github.com/pypa/pip/blob/e060970d51c5946beac8447eb95585d83019582d/src/pip/_internal/utils/logging.py#L37) they mention that importing colorama can cause several errors including `SystemError` and `ImportError`, so I just mimicked their pattern in the case that importing colorama fails. I can put a comment above the except statement clarifying this, or I can replace `Exception` with `(ImportError, SystemError) as e`.

At the moment, I don't have a way of testing this on Windows and what kind of exceptions we may need to catch, but I'm going to look at the colorama repo to see if they shed any light on this. I'm also working to implement #650 so that we can automatically test against Windows and macOS in our CI pipeline (in addition to Linux on Travis, though GitHub actions allows us to test against Linux as well).

Reply:
I don't follow why `colorama` wouldn't be available, given that it's listed in `install_requires`. I wonder if pip's exception handling has more to do with the fact that they've vendored colorama.

Also, if this were a common pattern, I would expect to see it in colorama's [usage guide](https://github.com/tartley/colorama#usage), or as an [issue](https://github.com/tartley/colorama/search?q=import+exception&type=Issues) when I search for "import exception".

Can we start without the exception handling, i.e. assume that if `pip install twine` works, then `import colorama` will work? That would make the `main()` implementation cleaner.